src/app/components/cds/charts/ListWidgetItem.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './ListWidget.module.css';
+
+const ListWidgetItem = ({
+  id,
+  itemLink,
+  itemText,
+  itemValue,
+}) => (
+  <li className={styles.listWidgetItemWrapper} key={id}>
+    {
+      itemLink ?
```

Comment:
suggestion: validate url in itemLink

Reply:
Don't worry too much about it, since we'll probably only have trusted backend generated links, not user input.
But it's a nice to have, if you got enough time for this.

Reply:
@amoedoamorim questions:

I tried both using `URL.canParse` and the helper `validateURL` and ran into different problems with both:
- `URL.canParse`: it renders, when manually testing it works, whether I send null an empty string or a malformed URL. But when I run the tests it fails `Uncaught [TypeError: URL.canParse is not a function]`
-  `validateURL`: it fails both when I test manually and run tests, with `Error: Uncaught [TypeError: Cannot read property 'length' of null]`

What the code looked like:
```
    {
      URL.canParse(itemLink) ?
        <div><a href={itemLink}>{itemText}</a></div>
        :
        <div className={styles.listWidgetItemText}>{itemText}</div>
    }
```

```
    {
      validateURL(itemLink) ?
        <div><a href={itemLink}>{itemText}</a></div>
        :
        <div className={styles.listWidgetItemText}>{itemText}</div>
    }
```

Reply:
I never user `URL.canParse`, I'm actually learning about it right now! :smile:
It seems it's a new Web API that's not implemented by the Node version that runs the unit tests, this is why it works on the browser and fails on the test.

I think `validateURL` expects a string as an argument. We can fix the helper function to return false if argument is not a string.

Reply:
> It seems it's a new Web API that's not implemented by the Node version that runs the unit tests, this is why it works on the browser and fails on the test.

Aaaaaaaaaaa! That makes sense! 😅 

> I think `validateURL` expects a string as an argument. We can fix the helper function to return false if argument is not a string.

That fixed it! But I thought it was better to create `isValidURL` that just returns a boolean? It runs `linkify.test(value)` instead of `match`. But let me know if you prefer that I change `validateURL`